### PR TITLE
Features/cicd api

### PR DESCRIPTION
**Pull request by Dercraker**

### CODE REVIEW

File: .github/workflows/docker-image.yml [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Setup .NET 8.0 SDK
+        uses: actions/setup-dotnet@v1
+        with:
+          dotnet-version: 8.x.x
+
+      - name: Cache .NET packages
+        uses: actions/cache@v2
```
Comment (BaptisteLegat) [inline .github/workflows/docker-image.yml]: Pas de v3 ou v4 ? https://github.com/actions/cache

**Review by BaptisteLegat — DISMISSED** [review]

**Comment by BaptisteLegat** [comment]
CI KO

File: .github/workflows/docker-image.yml [hunk 2]
```diff
@@ -0,0 +1,133 @@
+name: Build and Deploy From A2B API
+
+on:
+  push:
+    branches:
+      - main
+      - develop
```
Comment (Baptiste-Ferrand) [inline .github/workflows/docker-image.yml:7]: why on push and on PR ?

Comment (Dercraker) [reply .github/workflows/docker-image.yml:7]: for testing availability of new code to be build and deployed before merge

**Comment by Dercraker** [comment]
Pipeline Fix

**Review by BaptisteLegat — APPROVED** [review]

**Pull request merged**